### PR TITLE
chore(release): v10.3.0

### PR DESCRIPTION
**Pull request by joshblack**
v10.3.0 release 🎉

### CODE REVIEW

File: packages/react/src/__tests__/index-test.js
Comment (jnm2377) [inline packages/react/src/__tests__/index-test.js:54]: Just realized that `HeaderContainer` (new state manager) is missing, and it's probably my fault. I didn't include `export HeaderContainer from './HeaderContainer';` in `react/src/components/UIShell/index.js`. Is that the reason it's not be included with everything else in the `*`?

Comment (joshblack) [reply packages/react/src/__tests__/index-test.js:54]: No sweat! I bet that's why. Want to commit the update? I think updating it in the `UISHell/index.js` and then doing:

```
cd packages/react
yarn test -u
```

Will update the snapshot 👍

**Review by dakahn — APPROVED** [review]

**Review by emyarod — APPROVED** [review]
🎉

**Comment by aledavila** [comment]
its good to me. But did have a question for @designertyler:

Did we want to hold off on the switcher ?

**Comment by designertyler** [comment]
> its good to me. But did have a question for @designertyler:
> 
> Did we want to hold off on the switcher?

Are you referring to the rename of the switcher icon? Chris has a PR open (I've approved) but it needs another approval and to be merged. https://github.com/carbon-design-system/carbon/pull/2913#pullrequestreview-246193686

**Review by designertyler — APPROVED** [review]

**Pull request merged**
